dashboard/app/views/pd/fit_weekend1819_registration/invalid.html.haml

```diff
@@ -0,0 +1,10 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+
+%p
+  Invalid Application
+
+%p
+  This does not appear to be a link for a facilitator application; if you applied
+  both as a facilitator and a teacher, please make sure you have followed the
```

Comment:
Do we allow you to apply as both?

Reply:
Yep! Most commonly used by teachers already familiar with one program who want to both become facilitators for that program and learn another program